scene/gui/color_picker.cpp

```diff
@@ -1428,3 +1431,64 @@ void ColorPickerButton::_bind_methods() {
 ColorPickerButton::ColorPickerButton() {
 	set_toggle_mode(true);
 }
+
+/////////////////
+
+void ColorPresetButton::_notification(int p_what) {
+	switch (p_what) {
+		case NOTIFICATION_DRAW: {
+			const Rect2 r = Rect2(Point2(0, 0), get_size());
+			Ref<StyleBox> sb_raw = get_theme_stylebox(SNAME("preset_fg"), SNAME("ColorPresetButton"))->duplicate();
+
+			if (sb_raw->get_class_name() == SNAME("StyleBoxFlat")) {
```

Comment:
StyleBoxes are Resources, so you can assign-check them like InputEvents (see any input method), e.g.
```
Ref<StyleBoxFlat> sb_flat = sb_raw;
if (sb_raw.is_valid()) {
    ...
}
```

Reply:
Done.